src/helpers.ts

```diff
@@ -187,21 +220,18 @@ export function combineSOCObjects(SOCObjects) {
     return combined;
 }
 
-export async function queryWebsocMultiple(params, fieldName) {
-    let responses = [];
+export async function queryWebsocMultiple(params: {[key: string]: string}, fieldName: string) {
+    let responses: WebsocResponse[] = [];
```

Comment:
nit: this can be const right

Reply:
I guess there are tons of let everywhere so 🤷

Reply:
Just gonna leave them as `let`. We can fix them all at once by adding an eslint rule in the style PR.